egs/wsj/s5/steps/compute_cmvn_stats.sh

```diff
@@ -38,17 +38,23 @@ if [ "$1" == "--two-channel" ]; then
   two_channel=true
   shift
 fi
+if [ "$1" == "--out-suffix" ]; then
```

Comment:
There have been some changes to `make_mfcc.sh` and `compute_cmvn_stats.sh` this week. Ensure these don't conflict with your changes.